src/components/ChatMap/ChatMapStyled.js

```diff
@@ -1,8 +1,28 @@
 import styled from 'styled-components';
```

Comment:
Wrong formatting

Reply:
Can we leave this for task TAL-55 Search by map and I'll move this styles to ChatMapStyled and clear file?